### ISSUE TITLE
gorule-0000061 update implementation to reflect new rule wording updates

### ISSUE THREAD

**Issue by dougli1sqrd** [geneontology/go-site#1658]
The rule text has been updated, and so the implementation shall follow.

**Comment by dougli1sqrd** [comment]
If the interpretation above is to be taken (where we just repair, and no filtering) Is this just a superset of gorule-0000059 then, in fact?

In a call March 9, we said that this rule was different because we filter on `RO:0002325 "colocoalizes_with"` with `BFO:0000050`, but if instead we are repairing, we no longer have any filters in this rule, and only repairs, like gorule-0000059.

**Comment by pgaudet** [comment]
Hi @dougli1sqrd 

The formulation is probably not clear. We want the combination of `RO_0002325 "colocalizes_with"` and `"GO:0032991 "protein-containing complex"`to be filtered. 

How about if we formulated like this: 
> If the annotation is to `"GO:0032991 "protein-containing complex"` or a subclass descendant of, then the allowed gp2term relation is `"BFO:0000050 "part of"`. If the gp2term relation is `RO:0002432 "is_active_in"`. or `RO:0001025 "located in", is repaired to `"BFO:0000050 "part of"`". Else, the annotation is filtered. 

 Is that clearer?

**Comment by dougli1sqrd** [comment]
@pgaudet 
You're saying:
1. If annotation is to `subClassOf GO:0032991 "protein-containing complex"` with any of {`RO:0002432 "is_active_in"`, or `RO:0001025 "located in"`} the  repair relation to `BFO:0000050 "part of"`.
2. If annotation is to `subClassOf GO:0032991 "protein-containing complex"` with anything else then filter the annotation.

How come we can't repair everything that's not part of the set to part of?

But if what I stated above is what you mean, then yes, that's clear.

**Comment by dougli1sqrd** [comment]
@vanaukenk oh okay thanks! That makes sense. Yeah let's update the rule wording to something like what @pgaudet or I have written and I'll also update the implementation again. Maybe this is the round that makes it!

**Comment by dougli1sqrd** [comment]
And then just to be extra sure, annotations to another other term `subClassOf GO:0005575 "cellular component" not in subClassOf GO:0032991` and relation not in the allowed set {`RO:0001025 "located in"`, `RO:0002432 "is_active_in"`, `RO:0002325 "colocalizes_with"`} should be repaired to `RO:0001025 "located in"`.

No changes here that I'm aware of, just confirming my understanding of terms that aren't protein containing complex.

**Comment by vanaukenk** [comment]
Yes, other CC terms that are not equal to, or a subclass descendant of GO:0032991, and are not root (GO:0005575) can be repaired to 'located in'.

**Comment by pgaudet** [comment]
Now working as expected
